lib/tasks/base_path.rake

```diff
@@ -0,0 +1,24 @@
+namespace :base_path do
+  desc "Edits the base_path of a document, given its content_id and the new base_path"
+  task :edit, [:content_id, :base_path] => [:environment] do |_t, args|
+    content_id = args[:content_id]
+    base_path = args[:base_path]
+
+    begin
+      document = Document.find(content_id)
+    rescue DocumentFinder::RecordNotFound => e
+      puts "Error finding the document: #{e.inspect}"
+    end
+
+    if document
+      document.base_path = base_path
+      document.update_type = 'minor'
+      if document.save
+        puts "The #{document.class} with title \"#{document.title}\" has been successfully edited"
+        puts "Its new base_path is now \"#{document.base_path}\""
+      else
+        puts "Couldn't edit the #{document.class} with title \"#{document.title}\""
```

Comment:
If saving fails, there will probably be an associated validation error.  Perhaps use `document.save!`, which will give a stack trace if there's something wrong.

Reply:
don't think there's a `save!` method here - specialist publisher isn't a database backed application so this isn't a traditional ActiveRecord model.